akash/chain.json

```diff
@@ -27,27 +27,25 @@
   },
   "codebase": {
     "git_repo": "https://github.com/akash-network/node/",
-    "recommended_version": "v0.22.0",
+    "recommended_version": "v0.22.7",
     "compatible_versions": [
-      "v0.20.0"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/akash-network/node/releases/download/v0.22.0/akash_0.22.0_linux_amd64.zip",
-      "linux/arm64": "https://github.com/akash-network/node/releases/download/v0.22.0/akash_0.22.0_linux_arm64.zip"
+      "linux/amd64": "https://github.com/akash-network/node/releases/download/v0.22.7/akash_linux_amd64.zip",
+      "linux/arm64": "https://github.com/akash-network/node/releases/download/v0.22.7/akash_linux_arm64.zip"
     },
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/akash-network/net/master/mainnet/genesis.json"
     },
     "versions": [
       {
-        "name": "v0.22.0",
-        "recommended_version": "v0.22.0",
+        "name": "v0.22.7",
+        "recommended_version": "v0.22.7",
         "compatible_versions": [
-          "v0.20.0"
```

Comment:
Can we please retain and expand the compatible_versions arrays?

Reply:
at this moment there `v0.22.7` is the only version that is compatible

Reply:
oh, then, can we please create a new version object (so one for v.0.22.0, and another for v.0.22.7), and also include the version name in its own compatible versions array.